app/assets/javascripts/discourse/app/components/reviewable-topic-link.gjs

```diff
@@ -23,7 +23,11 @@ export default class ReviewableTopicLink extends Component {
             this.reviewable
           }}</a>
         {{categoryBadge this.reviewable.category}}
-        <ReviewableTags @tags={{this.reviewable.topic_tags}} @tagName="" />
+        <ReviewableTags
+          @topic={{this.reviewable.topic}}
+          @tags={{this.reviewable.topic_tags}}
+          @tagName=""
```

Comment:
if we always pass in `tagName=""` then perhaps the component can be converted to glimmer? (so it doesn't have a wrapping tag in the first place)